sycl/test-e2e/Basic/sycl_2020_images/sampled_image.cpp

```diff
@@ -0,0 +1,121 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %BE_RUN_PLACEHOLDER %t.out
+
+// Tests the members of the sampled_image class.
+
+#include "common.hpp"
+
+constexpr size_t IMAGE_X = 5;
```

Comment:
nit, these are used for creating ranges, so better names might be `IMAGE_WIDTH`, `IMAGE_HEIGHT` and `IMAGE_DEPTH`  .  
When working with images there are always too many terms in play, so it's good practice to use "width", "height", "depth" for bounds and "x", "y", "z" for coords